`TestPauseResumeCatchup` fails intermittently

https://github.com/filecoin-project/go-f3/actions/runs/9909586496/job/27377974698?pr=458

```
-test.shuffle 1720795699833660212
2024-07-12T14:48:22.399Z	ERROR	f3	go-f3/host.go:584	error while saving decision: saving ceritifcate in a store: putting the cert: FAILURE!
2024-07-12T14:48:28.728Z	ERROR	f3	go-f3/f3.go:97	failed to broadcast message: publishing message: validation ignored
--- FAIL: TestPauseResumeCatchup (36.12s)
    f3_test.go:327: 
        	Error Trace:	/home/runner/work/go-f3/go-f3/test/f3_test.go:327
        	            				/home/runner/work/go-f3/go-f3/test/f3_test.go:70
        	Error:      	Condition never satisfied
        	Test:       	TestPauseResumeCatchup
FAIL
```

```
2024-07-23T13:48:51.909Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:48:51.957Z	ERROR	f3	go-f3/host.go:595	error while saving decision: saving ceritifcate in a store: putting the cert: Intentional error for testing, please ignore!
2024-07-23T13:48:52.680Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:48:52.705Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:48:52.724Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:48:52.741Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:48:55.038Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
--- FAIL: TestF3PauseResumeCatchup (14.15s)
    f3_test.go:72: 
        	Error Trace:	/home/runner/work/go-f3/go-f3/test/f3_test.go:72
        	Error:      	"2" is not less than "2"
        	Test:       	TestF3PauseResumeCatchup
2024-07-23T13:49:21.535Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
2024-07-23T13:49:28.873Z	ERROR	f3	go-f3/f3.go:101	failed to broadcast message: publishing message: validation ignored
```

--- a/test/f3_test.go
+++ b/test/f3_test.go
@@ -60,8 +60,8 @@ func TestF3PauseResumeCatchup(t *testing.T) {
 
 	// Resuming node 1 should continue agreeing on instances.
 	env.resumeNode(1)
-	require.Equal(t, oldInstance, newInstance)
-	resumeInstance := newInstance + 1
+	// make sure we're ahead of node 2, which may be ahead of node 0.
+	resumeInstance := env.nodes[2].currentGpbftInstance() + 1
 	env.waitForInstanceNumber(resumeInstance, 30*time.Second, false)
 
 	// Wait until we're far enough that pure GPBFT catchup should be impossible.
@@ -69,6 +69,7 @@ func TestF3PauseResumeCatchup(t *testing.T) {
 	env.waitForInstanceNumber(targetInstance, 30*time.Second, false)
 
 	pausedInstance := env.nodes[2].currentGpbftInstance()
+
 	require.Less(t, pausedInstance, resumeInstance)
 
 	env.resumeNode(2)

